app/feature/feature-movingflow/src/main/kotlin/com/hedvig/android/feature/movingflow/data/MovingFlowQuotes.kt

```diff
@@ -0,0 +1,108 @@
+package com.hedvig.android.feature.movingflow.data
+
+import com.hedvig.android.core.uidata.UiMoney
+import com.hedvig.android.data.productVariant.android.toProductVariant
+import com.hedvig.android.data.productvariant.ProductVariant
+import com.hedvig.android.feature.movingflow.data.MovingFlowQuotes.DisplayItem
+import com.hedvig.android.feature.movingflow.data.MovingFlowQuotes.MoveHomeQuote
+import com.hedvig.android.feature.movingflow.data.MovingFlowQuotes.MoveHomeQuote.Deductible
+import com.hedvig.android.feature.movingflow.data.MovingFlowQuotes.MoveMtaQuote
+import kotlinx.datetime.LocalDate
+import kotlinx.serialization.Serializable
+import octopus.feature.movingflow.fragment.MoveIntentQuotesFragment
+import octopus.feature.movingflow.fragment.MoveQuoteDisplayItemFragment
+
+@Serializable
+internal data class MovingFlowQuotes(
+  val homeQuotes: List<MoveHomeQuote>,
+  val mtaQuotes: List<MoveMtaQuote>,
+) {
+  interface Quote {
```

Comment:
would we want to have this general Quote interface for TierDeductibleQuote to inherit from too later?

Reply:
On one hand it feels that way, on another I do see there are quite some differences between the three.
The reason I added an interface in this particular scenario was so that I could present them in one common way on the UI later. Other than that there are a bunch of things that are different between the three that I don't know if a common interface would work.
We may need to do it if we want to in the future share the UI between these two screens in moving and change tier flows, but we can look into that when we get there